### ISSUE TITLE
Q: Adding the value of a column to the label in the axis (ColumnChart, CategoryAxis)

### ISSUE THREAD

**Issue by Chnoch** [amcharts/amcharts4#726]
I'm trying to recreate the following image (beware that the numbers and column heights don't match, this is just a design proposition):
![image](https://user-images.githubusercontent.com/247564/50285940-02af7180-045e-11e9-8165-940e9f290446.png)

I'm using an XYChart with a CategoryAxis and a ColumnSeries. How can I add the value of each column to the label in the axis, with different style definitions (and a line break if the definition is in the same property)? 
Thanks for your help!

**Comment by martynasma** [comment]
Hm, good question :)

How about using `LabelBullet`, then positioning those using `y` adapter.

Just make sure to `chart.maskeBullets = false` the chart.

Example:
https://codepen.io/team/amcharts/pen/a5483a101af20200af1853a183fb6ec0?editors=0010

**Comment by RobbiesCode** [comment]
The label is automatically hidden if the column is not tall enough (even if the label is positioned outside).

To prevent this, add:

```
bullet1.label.hideOversized = false;
```

Hope this helps!

**Comment by Chnoch** [comment]
I do have a follow up question for the second issue I faced:

> Another issue I faced is that the we have a relatively small width for the columns (`series.columns.template.width = 15;`). This leads to an occlusion and ellipsis of parts of the bullet labels. You can try it out by setting `series.columns.template.width = 20;` in the codepen, with a value of `15` the labels are completely hidden. Can we also override the series to not mask the bullets?

This can be reproduced in https://codepen.io/chnoch/pen/MZGVEx, where the values are ellipsed. Can this be prevented?

Thanks to your comment @RobbiesCode, this solved my first issue.

**Comment by martynasma** [comment]
@Chnoch 

Bullet's have their `miWidth` automatically set to width of the column, hence bullet labels are truncated.

To fix simply do this:

```
bullet.label.truncate = false;
```

**Comment by martynasma** [comment]
@NPC No, there's no way to set up series without proper data fields.

That said, you don't need a series just for the labels. You can plop those labels on one of the "real" series, then position labelbullet with its `locationY`.

I modified your pen:
https://codepen.io/team/amcharts/pen/96b4073e48bcbb337dcd03666f9cea97

**Comment by NPC** [comment]
@martynasma That's also a good idea, I'll keep it in mind, but not suitable for cluster sub-category labels, since the label disappears if the series is hidden in legend. So I'll have to stick to the "fake" series solution.

Thanks for your help in reviewing it!
